packages/ember-application/lib/system/application.js

```diff
@@ -793,35 +978,198 @@ Application.reopenClass({
 if (isEnabled('ember-application-visit')) {
   Application.reopen({
     /**
-      Creates a new instance of the application and instructs it to route to the
-      specified initial URL. This method returns a promise that will be resolved
-      once rendering is complete. That promise is resolved with the instance.
-
-      ```js
-      App.visit('/users').then(function(instance) {
-        var view = instance.view;
-        view.appendTo('#qunit-test-fixtures');
-      });
-     ```
+      Boot a new instance of `Ember.ApplicationInstance` for the current
+      application and navigate it to the given `url`. Returns a `Promise` that
+      resolves with the instance when the initial routing and rendering is
+      complete, or rejects with any error that occured during the boot process.
 
-      @method visit
-      @private
-    */
-    visit(url) {
-      var instance = this.buildInstance();
-      this.runInstanceInitializers(instance);
-
-      var renderPromise = new Ember.RSVP.Promise(function(res, rej) {
-        instance.didCreateRootView = function(view) {
-          instance.view = view;
-          res(instance);
+      When `autoboot` is disabled, calling `visit` would first cause boot the
```

Comment:
I think the subject and verb are switched here? "would first cause the application to boot" instead?